actions/python-poetry-bump-version/action.yaml

```diff
@@ -30,7 +30,7 @@ runs:
   using: "composite"
   steps:
     - name: Set up Poetry version ${{ inputs.poetry-version }}
-      uses: bakdata/ci-templates/actions/python-setup-poetry@v1.0.0
+      uses: bakdata/ci-templates/actions/python-setup-poetry@v1.3.0
```

Comment:
Don't forget to create this tag ;)

Reply:
yep, will do after it's merged 🙂 hopefully we'll have a proper release pipeline for this soon